generation/WinSDK/enums.json

```diff
@@ -17836,24 +17869,31 @@
   },
   {
     "name": "FONT_PITCH_AND_FAMILY",
```

Comment:
This enum needs to be reviewed based on the following:
https://docs.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-logfontw

> The pitch and family of the font. The two low-order bits specify the pitch of the font and can be one of the following values.
> 
> DEFAULT_PITCH
> FIXED_PITCH
> VARIABLE_PITCH
> Bits 4 through 7 of the member specify the font family and can be one of the following values.
> FF_DECORATIVE
> FF_DONTCARE
> FF_MODERN
> FF_ROMAN
> FF_SCRIPT
> FF_SWISS
> The proper value can be obtained by using the Boolean OR operator to join one pitch constant with one family constant.

Reply:
Hm perhaps `lfPitchAndFamily`/`iPitchAndFamily` for LOGFONTW/CreateFontW shouldn't be an enum. Based on the docs it looks as if this should be kept as a `byte` and instead have enums  defined for `FF_` and `_PITCH` to specify PitchAndFamily.
@mikebattista what are your thoughts on this?

Reply:
How does WinForms handle this today? Does it use an enum here or just bit operations against constants?

Reply:
WinForms does not ever explicitly set a LOGFONTW's `lfPitchAndFamily`. We call to `Font.ToLogFont()` which returns a LOGFONTW and pass the `lfPitchAndFamily` value where needed. When we had our own definition of LOGFONTW we kept `lfPitchAndFamily` as just a `byte`.

Reply:
Ok. We should probably start with the existing WinForms model and @lonitra's recommendation then.